Mathlib/Algebra/MonoidAlgebra/MapDomain.lean

```diff
@@ -159,7 +152,7 @@ since the changes that have made `nsmul` definitional, this would be possible,
 but for now we just construct the ring isomorphisms using `RingEquiv.refl _`.
 -/
 
-
+variable (k G) in
```

Comment:
It might be cleaner to just do this once, at this point, in a section, rather than two local variable changes.

Reply:
I just tried, but I think that's clearly worse. It's longer and introduces a section for only two declarations and makes `variable`-introduced variables explicit (which is usually considered bad taste due to the effect-at-a-distance phenomenon)

Reply:
That's fair!